NOTICE

```diff
@@ -35,7 +35,7 @@ Copyright (C) 2013 GoDaddy Operating Company, LLC
 
 ~~~
 
-lib/cppapi developed by LinkedIn
+include/tscpp/api, src/tscpp/api developed by LinkedIn
```

Comment:
This included some unrelated cleanup of kruft leftover from lib/cppapi moving to include|src/tscpp/api .